bls/bls_test.go

```diff
@@ -0,0 +1,154 @@
+package bls_test
```

Comment:
You should add more or less the same tests we have in the rust library, so also when we're expecting to fail due the wrong key.